### ISSUE TITLE
Vim 8.1.0922 Test failure on macOS 10.14

### ISSUE THREAD

**Issue by chdiza** [vim/vim#3965]
```
From test_terminal.vim:
Found errors in Test_terminal_postponed_scrollback():
Run 1:
function RunTheTest[40]..Test_terminal_postponed_scrollback[17]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_01.dump.failed", "dumps/Test_terminal_01.dump")
function RunTheTest[40]..Test_terminal_postponed_scrollback[23]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_02.dump.failed", "dumps/Test_terminal_02.dump")
function RunTheTest[40]..Test_terminal_postponed_scrollback[28]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_03.dump.failed", "dumps/Test_terminal_03.dump")
Run 2:
function RunTheTest[40]..Test_terminal_postponed_scrollback[17]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_01.dump.failed", "dumps/Test_terminal_01.dump")
function RunTheTest[40]..Test_terminal_postponed_scrollback[23]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_02.dump.failed", "dumps/Test_terminal_02.dump")
function RunTheTest[40]..Test_terminal_postponed_scrollback[28]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_03.dump.failed", "dumps/Test_terminal_03.dump")
Flaky test failed too often, giving up
TEST FAILURE
make[2]: *** [report] Error 1
make[1]: *** [scripttests] Error 2
make: *** [test] Error 2
```

**Comment by brammool** [comment]
> From test_terminal.vim:
> Found errors in Test_terminal_postponed_scrollback():
> Run 1:
> function RunTheTest[40]..Test_terminal_postponed_scrollback[17]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_01.dump.failed", "dumps/Test_terminal_01.dump")
> function RunTheTest[40]..Test_terminal_postponed_scrollback[23]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_02.dump.failed", "dumps/Test_terminal_02.dump")
> function RunTheTest[40]..Test_terminal_postponed_scrollback[28]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_03.dump.failed", "dumps/Test_terminal_03.dump")
> Run 2:
> function RunTheTest[40]..Test_terminal_postponed_scrollback[17]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_01.dump.failed", "dumps/Test_terminal_01.dump")
> function RunTheTest[40]..Test_terminal_postponed_scrollback[23]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_02.dump.failed", "dumps/Test_terminal_02.dump")
> function RunTheTest[40]..Test_terminal_postponed_scrollback[28]..VerifyScreenDump line 20: See dump file difference: call term_dumpdiff("Test_terminal_03.dump.failed", "dumps/Test_terminal_03.dump")
> Flaky test failed too often, giving up
> TEST FAILURE
> make[2]: *** [report] Error 1
> make[1]: *** [scripttests] Error 2
> make: *** [test] Error 2
> ```

It appears to be a race condition.  It passes on my machine, thus it's
hard to know how to fix it.  Increasing the sleep might work, even
though it's not a good solution.

-- 
ARTHUR:    Be quiet!  I order you to shut up.
OLD WOMAN: Order, eh -- who does he think he is?
ARTHUR:    I am your king!
OLD WOMAN: Well, I didn't vote for you.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by ichizok** [comment]
This is resolved at v8.1.0925.
